docs/v0.2/features/powershell/azure-devops.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Scripts related to interacting with Azure DevOps"
+layout: default
+---
+
+# Azure DevOps
+
+This module provides the following capabilities:
+- [Setting a variable in an Azure DevOps pipeline](#setting-a-variable-in-an-azure-devops-pipeline)
+
```

Comment:
Missing a reference to the new feature:

- [Setting ARM outputs to Azure DevOps variable group](#setting-arm-outputs-to-azure-devops-variable-group)

Reply:
Ah, yes, good catch! Thanks.